docs/_templates/autosummary/base.rst

```diff
@@ -0,0 +1,5 @@
+{{ objname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. auto{{ objtype }}:: {{ objname }}
```

Comment:
Do we need these or are standard solutions ok?

Reply:
I could try removing them. However, we at least need `module.rst`. The rest, I'm not sure of.